Support diorite and emery

dmorgan81 opened the pull request.

dmorgan81 committed:
Support diorite and emery

matthewtole reviewed (changes requested):
Can you also update the demo app with these platforms?

dmorgan81 committed:
Update demo app to support diorite and emery

Ran pebble convert-project as well to switch from old appinfo.json to
package.json

dmorgan81 commented:
Diorite and emery support added to demo app. I went ahead and converted appinfo.json to package.json.
